src/front-end/front-end-interface/AddGameScreen.cs

```diff
@@ -6,25 +6,31 @@
 using System.Runtime.InteropServices;
 using System.Windows.Forms;
 using System.Windows.Forms.VisualStyles;
-using static Games_In_One_App.MainScreen;
+using static Games_In_One.MainScreen;
 
-namespace Games_In_One_App
+namespace Games_In_One
 {
     public partial class AddGameScreen : UserControl
-    {   
-        private RefreshListFunc refreshListFunc;
+    {
+        private MainScreen main;
 
         [DllImport("LinkFrontAndBack.dll")]
         public static extern void add([In] string game_name, [In] string game_path, [In] string image_path);
-        public AddGameScreen(RefreshListFunc refreshListFunc)
+        public AddGameScreen()
         {
             InitializeComponent();
-            this.refreshListFunc = refreshListFunc;
+        }
+
+        public AddGameScreen(MainScreen main)
```

Comment:
Can we get some comments on the public functions?

Reply:
Added comments to public functions.